docs/source/notebooks/sampling_conjugate_step.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>There is <a href="https://arviz-devs.github.io/arviz/api/generated/arviz.plot_ess.html" target="_blank">a built-in</a> <code>plot_ess</code> in ArviZ if you want even more in-depth goodies here ;)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>